app/src/main/res/layout/activity_main.xml

```diff
@@ -1,176 +1,177 @@
+						android:stateListAnimator="@null"
+						android:text="Swipes"
+						android:textAllCaps="false"
+						android:textColor="@color/blue"
+						android:textSize="14sp"
+						android:textStyle="normal" />
 
 					<Button
-							android:id="@+id/brb"
-							android:layout_width="65dp"
-							android:layout_height="35dp"
-							android:layout_marginLeft="6dp"
-							android:layout_marginTop="15dp"
-							android:layout_marginRight="6dp"
-							android:background="@drawable/button"
-							android:fontFamily="sans-serif-medium"
-							android:gravity="center_horizontal"
-							android:onClick="filterClick"
-							android:padding="8dp"
-							android:stateListAnimator="@null"
-							android:text="BRB"
-							android:textAllCaps="false"
-							android:textColor="@color/blue"
-							android:textSize="14sp"
-							android:textStyle="normal" />
+						android:id="@+id/brb"
+						android:layout_width="65dp"
+						android:layout_height="35dp"
+						android:layout_marginLeft="6dp"
+						android:layout_marginTop="15dp"
+						android:layout_marginRight="6dp"
+						android:background="@drawable/button"
+						android:fontFamily="sans-serif-medium"
+						android:gravity="center_horizontal"
+						android:onClick="filterClick"
+						android:padding="8dp"
+						android:stateListAnimator="@null"
+						android:text="BRB"
+						android:textAllCaps="false"
+						android:textColor="@color/blue"
+						android:textSize="14sp"
+						android:textStyle="normal" />
 				</LinearLayout>
 			</HorizontalScrollView>
 
 			<android.support.v7.widget.RecyclerView
-					android:id="@+id/cafe_list"
-					android:layout_width="match_parent"
-					android:layout_height="match_parent"
-					android:layout_below="@+id/scroll"
-					android:clipToPadding="false"
-					android:nestedScrollingEnabled="false"
-					android:onClick="filterClick"
-					android:visibility="gone" />
+				android:id="@+id/cafe_list"
+				android:layout_width="match_parent"
+				android:layout_height="match_parent"
+				android:layout_below="@+id/scroll"
+				android:clipToPadding="false"
+				android:nestedScrollingEnabled="false"
+				android:onClick="filterClick"
+				android:visibility="gone" />
 		</RelativeLayout>
 	</ScrollView>
 
 	<android.support.design.widget.BottomNavigationView
-			android:id="@+id/bottom_navigation"
-			android:layout_width="match_parent"
-			android:layout_height="wrap_content"
-			android:layout_alignParentBottom="true"
-			app:itemBackground="@color/white"
-			app:menu="@menu/bottom_nav_main" />
+		android:id="@+id/bottom_navigation"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:layout_alignParentBottom="true"
+		app:itemBackground="@color/white"
+		app:menu="@menu/bottom_nav_main" />
 
 	<RelativeLayout
-			android:id="@+id/relative_layout_splash"
-			android:layout_width="match_parent"
-			android:layout_height="match_parent">
+		android:id="@+id/relative_layout_splash"
```

Comment:
I think we can remove the 'include layout splash screen' from the xml and just make it appear programmatically when Main Activity begins.